config/secrets.yml

```diff
@@ -1,37 +0,0 @@
-# Be sure to restart your server when you modify this file.
-
-# Your secret key is used for verifying the integrity of signed cookies.
-# If you change this key, all old signed cookies will become invalid!
-
-# Make sure the secret is at least 30 characters and all random,
-# no regular words or you'll be exposed to dictionary attacks.
-# You can use `rails secret` to generate a secure secret key.
-
-# Make sure the secrets in this file are kept private
-# if you're sharing your code publicly.
-
-# Shared secrets are available across all environments.
-
-# shared:
-#   api_key: a1B2c3D4e5F6
-
-# Environmental secrets are only available for that specific environment.
-shared:
-  authentication:
-    token: <%= ENV['ZYNC_AUTHENTICATION_TOKEN'] %>
-
-development:
-  secret_key_base: 7236814be2201361bc2ce32c3bb4e425ace58925f9967ed496f966e0fc835749ee30ce43b31a1a193f48c56a5c3eb7dbb05183b0415b829960f42f7175c0d4f4
-
-test:
-  secret_key_base: d1659d6e2beb5fff1876e32b778d89c32d1d54dfb0f6a474a56cb70166ce5771adb49957816c297f67f832d59d06d74f56bd9ab1a06a91136424c0f8b001c2db
```

Comment:
did you remove this file and adjusted circleci/config to avoid storing these here? There is also the `secret_key_base_dummy` option that can be used.